presto-docs/src/main/sphinx/functions/binary.rst

```diff
@@ -86,3 +86,19 @@ Binary Functions
 .. function:: xxhash64(binary) -> varbinary
 
     Computes the xxhash64 hash of ``binary``.
+
+.. function:: hmac_md5(binary, key) -> varbinary
+
+    Computes the HMAC MD5 of ``binary`` with a given ``key``.
```

Comment:
`MD5` -> `md5` to be consistent with the rest of the documentation. Similarly, please use lowercase for the other hash functions `sha1`, `sha256`, etc.